src/eth/index.ts

```diff
@@ -62,20 +62,24 @@ export class OpenSeaClient {
   readonly apiKey: string = ''
   readonly assetLimit: number = 50
   readonly eventLimit: number = 300
+  private fetchOptions = {};
 
   constructor(props?: OpenSeaClientProps) {
     this.url = props?.apiEndpoint ?? this.url
     this.apiKey = props?.apiKey ?? this.apiKey
     this.assetLimit = props?.assetLimit ?? this.assetLimit
     this.eventLimit = props?.eventLimit ?? this.eventLimit
+    if(this.apiKey) {
+      this.fetchOptions['X-API-KEY'] = this.apiKey
+    }
```

Comment:
```suggestion
  if(this.apiKey) {
    this.fetchOptions = {
      headers: new Headers({ 'X-API-KEY': this.apiKey })
    }
  }
```

Small change: api key needs to be set to a header within a headers prop on the request options

Reply:
Will make a new PR to get this out today. Closing in favor of https://github.com/AudiusProject/fetch-nft/pull/16

Reply:
oh yea good catch lol, i did a last second refactor and evidently tripped over myself :sweat_smile: